tests/test_io.py

```diff
@@ -95,6 +95,7 @@ def test_json_bytes_no_delim_defined(ndjson_points_file: str) -> None:
 
 
 def test_to_and_from_dask_array(daa: dak.Array) -> None:
+    daa = dak.from_awkward(daa.compute(), npartitions=3)
```

Comment:
the normal `daa` fixture is an array that comes from parquet, and the form ends up getting a mask when we do column selection. We avoid the mask by going ahead and computing the parquet-originating collection and creating a new collection from the concrete result via `from_awkward`. A bit roundabout but we still test what we mean to test here.